### DISCUSSION TITLE
Weaver java registry examples

### DISCUSSION THREAD

**Discussion by swapdonkey** [open-telemetry/opentelemetry-java#6908] [Q&A] [answered]
Hi,
We're looking to standardise some Semantic Conventions particular to our business across our enterprise. It looks like the OTEL Weaver project would be the way to go https://github.com/open-telemetry/weaver.
I've been trying to find examples or actual implementation of Java registry we can base ours on but I've been unable to find one. Does anyone have any links to an example?

**Comment by trask** [comment, marked as answer]
hi @swapdonkey, I'd suggest asking in https://github.com/open-telemetry/semantic-conventions

**Reply by swapdonkey** [reply]
Ok thanks @trask I'll do that

**Comment by swapdonkey** [comment]
Actually I just found the java registry here https://github.com/open-telemetry/semantic-conventions-java/blob/main/buildscripts/templates/registry/java/weaver.yaml
